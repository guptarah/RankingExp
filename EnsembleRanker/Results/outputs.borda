training model for iter ... 0
at noisy annotor id: 0
at noisy annotor id: 1
at noisy annotor id: 2
at noisy annotor id: 3
at noisy annotor id: 4
at noisy annotor id: 5
Model Trained.
Results:
Annotator specific results for annotator 0
TEST: Kendall Tau: 0.484198, Spearman correlation: 0.613860, Pearson correlation: 0.590057
DEV: Kendall Tau: 0.392507, Spearman correlation: 0.501555, Pearson correlation: 0.472913
Annotator specific results for annotator 1
TEST: Kendall Tau: 0.474396, Spearman correlation: 0.602988, Pearson correlation: 0.566146
DEV: Kendall Tau: 0.372146, Spearman correlation: 0.474591, Pearson correlation: 0.461563
Annotator specific results for annotator 2
TEST: Kendall Tau: 0.472322, Spearman correlation: 0.595588, Pearson correlation: 0.558579
DEV: Kendall Tau: 0.371703, Spearman correlation: 0.470067, Pearson correlation: 0.464872
Annotator specific results for annotator 3
TEST: Kendall Tau: 0.460257, Spearman correlation: 0.581068, Pearson correlation: 0.564060
DEV: Kendall Tau: 0.377457, Spearman correlation: 0.480156, Pearson correlation: 0.455924
Annotator specific results for annotator 4
TEST: Kendall Tau: 0.467420, Spearman correlation: 0.591588, Pearson correlation: 0.566448
DEV: Kendall Tau: 0.362408, Spearman correlation: 0.463095, Pearson correlation: 0.440049
Annotator specific results for annotator 5
TEST: Kendall Tau: 0.485141, Spearman correlation: 0.608089, Pearson correlation: 0.581790
DEV: Kendall Tau: 0.375023, Spearman correlation: 0.480572, Pearson correlation: 0.437421

Borda count results
TEST: Kendall Tau: 0.487403, Spearman correlation: 0.614883, Pearson correlation: 0.584477
DEV: Kendall Tau: 0.384982, Spearman correlation: 0.489079, Pearson correlation: 0.468524
training model for iter ... 1
at noisy annotor id: 0
at noisy annotor id: 1
at noisy annotor id: 2
at noisy annotor id: 3
at noisy annotor id: 4
at noisy annotor id: 5
Model Trained.
Results:
Annotator specific results for annotator 0
TEST: Kendall Tau: 0.389408, Spearman correlation: 0.497902, Pearson correlation: 0.466805
DEV: Kendall Tau: 0.412614, Spearman correlation: 0.534472, Pearson correlation: 0.491419
Annotator specific results for annotator 1
TEST: Kendall Tau: 0.349571, Spearman correlation: 0.446934, Pearson correlation: 0.423447
DEV: Kendall Tau: 0.358325, Spearman correlation: 0.462787, Pearson correlation: 0.417041
Annotator specific results for annotator 2
TEST: Kendall Tau: 0.307521, Spearman correlation: 0.394899, Pearson correlation: 0.358870
DEV: Kendall Tau: 0.317991, Spearman correlation: 0.411947, Pearson correlation: 0.358591
Annotator specific results for annotator 3
TEST: Kendall Tau: 0.370375, Spearman correlation: 0.471325, Pearson correlation: 0.461954
DEV: Kendall Tau: 0.393383, Spearman correlation: 0.506262, Pearson correlation: 0.438393
Annotator specific results for annotator 4
TEST: Kendall Tau: 0.249978, Spearman correlation: 0.324324, Pearson correlation: 0.290147
DEV: Kendall Tau: 0.250428, Spearman correlation: 0.322798, Pearson correlation: 0.292017
Annotator specific results for annotator 5
TEST: Kendall Tau: 0.379670, Spearman correlation: 0.486447, Pearson correlation: 0.445736
DEV: Kendall Tau: 0.387086, Spearman correlation: 0.503186, Pearson correlation: 0.463519

Borda count results
TEST: Kendall Tau: 0.365949, Spearman correlation: 0.469569, Pearson correlation: 0.439546
DEV: Kendall Tau: 0.374323, Spearman correlation: 0.488182, Pearson correlation: 0.447697
training model for iter ... 2
at noisy annotor id: 0
at noisy annotor id: 1
at noisy annotor id: 2
at noisy annotor id: 3
at noisy annotor id: 4
at noisy annotor id: 5
Model Trained.
Results:
Annotator specific results for annotator 0
TEST: Kendall Tau: 0.424527, Spearman correlation: 0.546421, Pearson correlation: 0.507046
DEV: Kendall Tau: 0.389292, Spearman correlation: 0.503628, Pearson correlation: 0.495436
Annotator specific results for annotator 1
TEST: Kendall Tau: 0.356100, Spearman correlation: 0.460799, Pearson correlation: 0.418381
DEV: Kendall Tau: 0.301465, Spearman correlation: 0.391503, Pearson correlation: 0.374515
Annotator specific results for annotator 2
TEST: Kendall Tau: 0.330074, Spearman correlation: 0.425360, Pearson correlation: 0.388194
DEV: Kendall Tau: 0.263257, Spearman correlation: 0.343610, Pearson correlation: 0.333105
Annotator specific results for annotator 3
TEST: Kendall Tau: 0.360197, Spearman correlation: 0.468351, Pearson correlation: 0.410264
DEV: Kendall Tau: 0.314677, Spearman correlation: 0.410187, Pearson correlation: 0.405163
Annotator specific results for annotator 4
TEST: Kendall Tau: 0.201414, Spearman correlation: 0.259889, Pearson correlation: 0.236484
DEV: Kendall Tau: 0.095073, Spearman correlation: 0.122810, Pearson correlation: 0.143111
Annotator specific results for annotator 5
TEST: Kendall Tau: 0.385725, Spearman correlation: 0.501927, Pearson correlation: 0.463754
DEV: Kendall Tau: 0.350206, Spearman correlation: 0.453148, Pearson correlation: 0.442451

Borda count results
TEST: Kendall Tau: 0.372110, Spearman correlation: 0.485240, Pearson correlation: 0.449749
DEV: Kendall Tau: 0.313963, Spearman correlation: 0.408956, Pearson correlation: 0.411915
training model for iter ... 3
at noisy annotor id: 0
at noisy annotor id: 1
at noisy annotor id: 2
at noisy annotor id: 3
at noisy annotor id: 4
at noisy annotor id: 5
Model Trained.
Results:
Annotator specific results for annotator 0
TEST: Kendall Tau: 0.316462, Spearman correlation: 0.410551, Pearson correlation: 0.401258
DEV: Kendall Tau: 0.331456, Spearman correlation: 0.432243, Pearson correlation: 0.444187
Annotator specific results for annotator 1
TEST: Kendall Tau: 0.300215, Spearman correlation: 0.388912, Pearson correlation: 0.373276
DEV: Kendall Tau: 0.311803, Spearman correlation: 0.409909, Pearson correlation: 0.425782
Annotator specific results for annotator 2
TEST: Kendall Tau: 0.373238, Spearman correlation: 0.479983, Pearson correlation: 0.490439
DEV: Kendall Tau: 0.364278, Spearman correlation: 0.468292, Pearson correlation: 0.456612
Annotator specific results for annotator 3
TEST: Kendall Tau: 0.344136, Spearman correlation: 0.444054, Pearson correlation: 0.440178
DEV: Kendall Tau: 0.355040, Spearman correlation: 0.459450, Pearson correlation: 0.477637
Annotator specific results for annotator 4
TEST: Kendall Tau: 0.358419, Spearman correlation: 0.458450, Pearson correlation: 0.426188
DEV: Kendall Tau: 0.276033, Spearman correlation: 0.350684, Pearson correlation: 0.247007
Annotator specific results for annotator 5
TEST: Kendall Tau: 0.342886, Spearman correlation: 0.444849, Pearson correlation: 0.425304
DEV: Kendall Tau: 0.358578, Spearman correlation: 0.463752, Pearson correlation: 0.470344

Borda count results
TEST: Kendall Tau: 0.358598, Spearman correlation: 0.463388, Pearson correlation: 0.455184
DEV: Kendall Tau: 0.356809, Spearman correlation: 0.460482, Pearson correlation: 0.450636
training model for iter ... 4
at noisy annotor id: 0
at noisy annotor id: 1
at noisy annotor id: 2
at noisy annotor id: 3
at noisy annotor id: 4
at noisy annotor id: 5
Model Trained.
Results:
Annotator specific results for annotator 0
TEST: Kendall Tau: 0.334011, Spearman correlation: 0.436867, Pearson correlation: 0.452177
DEV: Kendall Tau: 0.376228, Spearman correlation: 0.465186, Pearson correlation: 0.506470
Annotator specific results for annotator 1
TEST: Kendall Tau: 0.324971, Spearman correlation: 0.425002, Pearson correlation: 0.431154
DEV: Kendall Tau: 0.384601, Spearman correlation: 0.474383, Pearson correlation: 0.522863
Annotator specific results for annotator 2
TEST: Kendall Tau: 0.344034, Spearman correlation: 0.435635, Pearson correlation: 0.414102
DEV: Kendall Tau: 0.391657, Spearman correlation: 0.489287, Pearson correlation: 0.499696
Annotator specific results for annotator 3
TEST: Kendall Tau: 0.353272, Spearman correlation: 0.459846, Pearson correlation: 0.474493
DEV: Kendall Tau: 0.394649, Spearman correlation: 0.486250, Pearson correlation: 0.526368
Annotator specific results for annotator 4
TEST: Kendall Tau: 0.294704, Spearman correlation: 0.386597, Pearson correlation: 0.420102
DEV: Kendall Tau: 0.352598, Spearman correlation: 0.434276, Pearson correlation: 0.479221
Annotator specific results for annotator 5
TEST: Kendall Tau: 0.357202, Spearman correlation: 0.462015, Pearson correlation: 0.467157
DEV: Kendall Tau: 0.416791, Spearman correlation: 0.516927, Pearson correlation: 0.563459

Borda count results
TEST: Kendall Tau: 0.361526, Spearman correlation: 0.468829, Pearson correlation: 0.467274
DEV: Kendall Tau: 0.401161, Spearman correlation: 0.494634, Pearson correlation: 0.533442
training model for iter ... 5
at noisy annotor id: 0
at noisy annotor id: 1
at noisy annotor id: 2
at noisy annotor id: 3
at noisy annotor id: 4
at noisy annotor id: 5
Model Trained.
Results:
Annotator specific results for annotator 0
TEST: Kendall Tau: 0.419582, Spearman correlation: 0.519945, Pearson correlation: 0.557523
DEV: Kendall Tau: 0.356416, Spearman correlation: 0.448815, Pearson correlation: 0.476877
Annotator specific results for annotator 1
TEST: Kendall Tau: 0.408046, Spearman correlation: 0.502639, Pearson correlation: 0.548094
DEV: Kendall Tau: 0.321014, Spearman correlation: 0.404287, Pearson correlation: 0.418215
Annotator specific results for annotator 2
TEST: Kendall Tau: 0.361529, Spearman correlation: 0.449855, Pearson correlation: 0.480392
DEV: Kendall Tau: 0.307894, Spearman correlation: 0.387762, Pearson correlation: 0.414810
Annotator specific results for annotator 3
TEST: Kendall Tau: 0.394463, Spearman correlation: 0.483922, Pearson correlation: 0.524015
DEV: Kendall Tau: 0.348294, Spearman correlation: 0.437358, Pearson correlation: 0.474240
Annotator specific results for annotator 4
TEST: Kendall Tau: 0.236678, Spearman correlation: 0.294181, Pearson correlation: 0.337802
DEV: Kendall Tau: 0.227510, Spearman correlation: 0.287289, Pearson correlation: 0.293115
Annotator specific results for annotator 5
TEST: Kendall Tau: 0.411380, Spearman correlation: 0.509270, Pearson correlation: 0.560393
DEV: Kendall Tau: 0.334758, Spearman correlation: 0.420435, Pearson correlation: 0.435673

Borda count results
TEST: Kendall Tau: 0.391300, Spearman correlation: 0.482711, Pearson correlation: 0.529674
DEV: Kendall Tau: 0.329135, Spearman correlation: 0.413960, Pearson correlation: 0.448714
training model for iter ... 6
at noisy annotor id: 0
at noisy annotor id: 1
at noisy annotor id: 2
at noisy annotor id: 3
at noisy annotor id: 4
at noisy annotor id: 5
Model Trained.
Results:
Annotator specific results for annotator 0
TEST: Kendall Tau: 0.351834, Spearman correlation: 0.443069, Pearson correlation: 0.461720
DEV: Kendall Tau: 0.391301, Spearman correlation: 0.493430, Pearson correlation: 0.513868
Annotator specific results for annotator 1
TEST: Kendall Tau: 0.300814, Spearman correlation: 0.380555, Pearson correlation: 0.391644
DEV: Kendall Tau: 0.379864, Spearman correlation: 0.477470, Pearson correlation: 0.482228
Annotator specific results for annotator 2
TEST: Kendall Tau: 0.315365, Spearman correlation: 0.397301, Pearson correlation: 0.416521
DEV: Kendall Tau: 0.369808, Spearman correlation: 0.470127, Pearson correlation: 0.473135
Annotator specific results for annotator 3
TEST: Kendall Tau: 0.361622, Spearman correlation: 0.454456, Pearson correlation: 0.485022
DEV: Kendall Tau: 0.397808, Spearman correlation: 0.494872, Pearson correlation: 0.507865
Annotator specific results for annotator 4
TEST: Kendall Tau: 0.257279, Spearman correlation: 0.323681, Pearson correlation: 0.335790
DEV: Kendall Tau: 0.234741, Spearman correlation: 0.296954, Pearson correlation: 0.300311
Annotator specific results for annotator 5
TEST: Kendall Tau: 0.334550, Spearman correlation: 0.418887, Pearson correlation: 0.437925
DEV: Kendall Tau: 0.376907, Spearman correlation: 0.473726, Pearson correlation: 0.493458

Borda count results
TEST: Kendall Tau: 0.332454, Spearman correlation: 0.418897, Pearson correlation: 0.452239
DEV: Kendall Tau: 0.374935, Spearman correlation: 0.473648, Pearson correlation: 0.498758
training model for iter ... 7
at noisy annotor id: 0
at noisy annotor id: 1
at noisy annotor id: 2
at noisy annotor id: 3
at noisy annotor id: 4
at noisy annotor id: 5
Model Trained.
Results:
Annotator specific results for annotator 0
TEST: Kendall Tau: 0.390315, Spearman correlation: 0.492262, Pearson correlation: 0.513328
DEV: Kendall Tau: 0.373732, Spearman correlation: 0.477477, Pearson correlation: 0.471835
Annotator specific results for annotator 1
TEST: Kendall Tau: 0.364682, Spearman correlation: 0.458974, Pearson correlation: 0.458831
DEV: Kendall Tau: 0.299651, Spearman correlation: 0.380954, Pearson correlation: 0.356637
Annotator specific results for annotator 2
TEST: Kendall Tau: 0.376907, Spearman correlation: 0.476129, Pearson correlation: 0.483017
DEV: Kendall Tau: 0.354191, Spearman correlation: 0.452838, Pearson correlation: 0.450319
Annotator specific results for annotator 3
TEST: Kendall Tau: 0.386963, Spearman correlation: 0.482463, Pearson correlation: 0.501956
DEV: Kendall Tau: 0.343240, Spearman correlation: 0.439801, Pearson correlation: 0.436676
Annotator specific results for annotator 4
TEST: Kendall Tau: 0.404906, Spearman correlation: 0.504486, Pearson correlation: 0.504670
DEV: Kendall Tau: 0.322841, Spearman correlation: 0.411824, Pearson correlation: 0.412927
Annotator specific results for annotator 5
TEST: Kendall Tau: 0.375526, Spearman correlation: 0.472252, Pearson correlation: 0.491889
DEV: Kendall Tau: 0.348394, Spearman correlation: 0.447111, Pearson correlation: 0.432471

Borda count results
TEST: Kendall Tau: 0.392287, Spearman correlation: 0.493052, Pearson correlation: 0.509236
DEV: Kendall Tau: 0.353977, Spearman correlation: 0.452180, Pearson correlation: 0.446866
training model for iter ... 8
at noisy annotor id: 0
at noisy annotor id: 1
at noisy annotor id: 2
at noisy annotor id: 3
at noisy annotor id: 4
at noisy annotor id: 5
Model Trained.
Results:
Annotator specific results for annotator 0
TEST: Kendall Tau: 0.373286, Spearman correlation: 0.477335, Pearson correlation: 0.471982
DEV: Kendall Tau: 0.242593, Spearman correlation: 0.312141, Pearson correlation: 0.272425
Annotator specific results for annotator 1
TEST: Kendall Tau: 0.309730, Spearman correlation: 0.394843, Pearson correlation: 0.377919
DEV: Kendall Tau: 0.164405, Spearman correlation: 0.208334, Pearson correlation: 0.136485
Annotator specific results for annotator 2
TEST: Kendall Tau: 0.261644, Spearman correlation: 0.336345, Pearson correlation: 0.347002
DEV: Kendall Tau: 0.137357, Spearman correlation: 0.180870, Pearson correlation: 0.183317
Annotator specific results for annotator 3
TEST: Kendall Tau: 0.345388, Spearman correlation: 0.441425, Pearson correlation: 0.440211
DEV: Kendall Tau: 0.196737, Spearman correlation: 0.251490, Pearson correlation: 0.204572
Annotator specific results for annotator 4
TEST: Kendall Tau: 0.162655, Spearman correlation: 0.203088, Pearson correlation: 0.212558
DEV: Kendall Tau: 0.055365, Spearman correlation: 0.070705, Pearson correlation: 0.068867
Annotator specific results for annotator 5
TEST: Kendall Tau: 0.348179, Spearman correlation: 0.446846, Pearson correlation: 0.431939
DEV: Kendall Tau: 0.185326, Spearman correlation: 0.240654, Pearson correlation: 0.206824

Borda count results
TEST: Kendall Tau: 0.326706, Spearman correlation: 0.418991, Pearson correlation: 0.415206
DEV: Kendall Tau: 0.180465, Spearman correlation: 0.233905, Pearson correlation: 0.199154
training model for iter ... 9
at noisy annotor id: 0
at noisy annotor id: 1
at noisy annotor id: 2
at noisy annotor id: 3
at noisy annotor id: 4
at noisy annotor id: 5
Model Trained.
Results:
Annotator specific results for annotator 0
TEST: Kendall Tau: 0.237099, Spearman correlation: 0.305406, Pearson correlation: 0.278517
DEV: Kendall Tau: 0.472510, Spearman correlation: 0.599243, Pearson correlation: 0.586271
Annotator specific results for annotator 1
TEST: Kendall Tau: 0.165885, Spearman correlation: 0.209297, Pearson correlation: 0.123504
DEV: Kendall Tau: 0.436676, Spearman correlation: 0.552618, Pearson correlation: 0.537932
Annotator specific results for annotator 2
TEST: Kendall Tau: 0.224420, Spearman correlation: 0.287313, Pearson correlation: 0.248925
DEV: Kendall Tau: 0.461577, Spearman correlation: 0.584578, Pearson correlation: 0.563378
Annotator specific results for annotator 3
TEST: Kendall Tau: 0.206669, Spearman correlation: 0.261223, Pearson correlation: 0.216295
DEV: Kendall Tau: 0.473076, Spearman correlation: 0.594959, Pearson correlation: 0.576567
Annotator specific results for annotator 4
TEST: Kendall Tau: 0.240691, Spearman correlation: 0.300033, Pearson correlation: 0.224302
DEV: Kendall Tau: 0.480616, Spearman correlation: 0.607842, Pearson correlation: 0.563898
Annotator specific results for annotator 5
TEST: Kendall Tau: 0.194624, Spearman correlation: 0.251979, Pearson correlation: 0.212813
DEV: Kendall Tau: 0.486272, Spearman correlation: 0.608969, Pearson correlation: 0.581807

Borda count results
TEST: Kendall Tau: 0.228435, Spearman correlation: 0.291141, Pearson correlation: 0.231783
DEV: Kendall Tau: 0.487007, Spearman correlation: 0.612607, Pearson correlation: 0.585781
