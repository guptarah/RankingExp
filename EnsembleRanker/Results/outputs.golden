training model for iter ... 0
Model Trained.
Results:
TEST: Kendall Tau: 0.501919, Spearman correlation: 0.634028, Pearson correlation: 0.601659
DEV: Kendall Tau: 0.386310, Spearman correlation: 0.497078, Pearson correlation: 0.454157
---------------------

training model for iter ... 1
Model Trained.
Results:
TEST: Kendall Tau: 0.384761, Spearman correlation: 0.491403, Pearson correlation: 0.458025
DEV: Kendall Tau: 0.422145, Spearman correlation: 0.544342, Pearson correlation: 0.507155
---------------------

training model for iter ... 2
Model Trained.
Results:
TEST: Kendall Tau: 0.418741, Spearman correlation: 0.540799, Pearson correlation: 0.502673
DEV: Kendall Tau: 0.366275, Spearman correlation: 0.472938, Pearson correlation: 0.470505
---------------------

training model for iter ... 3
Model Trained.
Results:
TEST: Kendall Tau: 0.324318, Spearman correlation: 0.422161, Pearson correlation: 0.415116
DEV: Kendall Tau: 0.350520, Spearman correlation: 0.458296, Pearson correlation: 0.466377
---------------------

training model for iter ... 4
Model Trained.
Results:
TEST: Kendall Tau: 0.360150, Spearman correlation: 0.469932, Pearson correlation: 0.468246
DEV: Kendall Tau: 0.415303, Spearman correlation: 0.514224, Pearson correlation: 0.557619
---------------------

training model for iter ... 5
Model Trained.
Results:
TEST: Kendall Tau: 0.422187, Spearman correlation: 0.522497, Pearson correlation: 0.565022
DEV: Kendall Tau: 0.332675, Spearman correlation: 0.417810, Pearson correlation: 0.423836
---------------------

training model for iter ... 6
Model Trained.
Results:
TEST: Kendall Tau: 0.322263, Spearman correlation: 0.406165, Pearson correlation: 0.409896
DEV: Kendall Tau: 0.394061, Spearman correlation: 0.497461, Pearson correlation: 0.499806
---------------------

training model for iter ... 7
Model Trained.
Results:
TEST: Kendall Tau: 0.409836, Spearman correlation: 0.514686, Pearson correlation: 0.518137
DEV: Kendall Tau: 0.385542, Spearman correlation: 0.493183, Pearson correlation: 0.473591
---------------------

training model for iter ... 8
Model Trained.
Results:
TEST: Kendall Tau: 0.378241, Spearman correlation: 0.483709, Pearson correlation: 0.471769
DEV: Kendall Tau: 0.249566, Spearman correlation: 0.319760, Pearson correlation: 0.264028
---------------------

training model for iter ... 9
Model Trained.
Results:
TEST: Kendall Tau: 0.257808, Spearman correlation: 0.330249, Pearson correlation: 0.275066
DEV: Kendall Tau: 0.507385, Spearman correlation: 0.637242, Pearson correlation: 0.604071
---------------------

